src/server/index.js

```diff
@@ -272,36 +282,14 @@ nextApp.prepare().then(() => {
     res.send(backing);
   });
 
-  server.get('/:id/file/backing.json', async (req, res) => {
-    if (!req.params.id) {
-      return res.status(400).send('Please provide the file key');
-    }
-    let data;
-
-    try {
-      data = await getFiles(req.params.id);
-
-      if (!data) {
-        return res.status(404).send('No file found');
-      }
-
-      const { recommendations } = await getFilesData(data);
-      const backing = getDependenciesAvailableForBacking(recommendations);
-      return res.status(200).send(backing);
-    } catch (err) {
-      console.error(err);
-      return res.status(400).send('Unable to fetch file');
-    }
-  });
-
-  server.get('/:id/profile/backing.json', async (req, res) => {
+  server.get('/:id/:type/backing.json', async (req, res) => {
```

Comment:
What is `:type` used for?

Reply:
We needed it before but since reverting back to saving both private and public files on s3, we don't need it at this point but I'm skeptical about removing it because we might need it in the future and it will be difficult to add it again when we subscriptions depending on the endpoint for dispatch.

Reply:
@znarf Also if I remove it here, we'll have a duplicate endpoint with https://github.com/opencollective/backyourstack/blob/feat%2Fselect-repos/src/server/index.js#L255